test/test.js

```diff
@@ -79,6 +79,52 @@ describe('memoize-fs', function () {
       })
     })
 
+    it('should be able to pass custom serialize and deserialize through memoizeFS options', (done) => {
+      const cachePath = path.join(__dirname, '../build/cache')
+      fs.removeSync(cachePath)
+
+      const {serialize, unserialize: deserialize} = serializer
```

Comment:
Shouldn’t this be `const {serialize, deserialize} = serializer`?

Reply:
`node-serialize` exports `serialize` and `unserialize`, i'm just renaming it to `deserialize` because it sounds more correct to me.